spec/factories/dialog_field.rb

```diff
@@ -1,6 +1,7 @@
 FactoryGirl.define do
   factory :dialog_field do
-    name "Dialog Field"
+    sequence(:name) { |n| "Dialog Field #{seq_padded_for_sorting(n)}" }
```

Comment:
The model doesn't do any validation on uniqueness of names - so this `sequence` shouldn't be necessary.